spec/features/content_pages_spec.rb

```diff
@@ -28,9 +28,44 @@ def content_urls
   end
 
   content_urls.each do |url|
-    scenario "visiting #{url}" do
-      visit url
-      expect(page).to have_http_status :success
+    describe url do
+      let(:url) { url }
+      before { visit(url) }
+      let(:document) { Nokogiri.parse(page.body) }
+
+      scenario "visiting #{url}", content: true do
```

Comment:
Can this be renamed to to avoid the double reference in an error (`describe` and `scenario` will both list the URL) - eg `responds with success`